bundles/org.openhab.binding.blink/README.md

```diff
@@ -0,0 +1,94 @@
+
+blink has implemented a 2-factor-authentication, so after the first login, a email or text message (as configured in
+your app settings) will be sent to you. This pin code needs to be entered into a form generated for each blink account
+bridge.
+
+The URL for pin verification is `<youropenhaburl>/blink/<accountUID>`. The easiest way to get this, is to copy the
+validationURL thing property:
+
+![](doc/verification-url.png)
+
+![](doc/2fa.png)
+
+### blink Camera
+
+One single blink camera, belonging to a blink network (see below).
+
+Configuration parameters should be set by auto-discovery after configuration of an account exclusively (i.e. no manual
+setup). For completeness, the configuration parameters set by auto-discovery are:
+
+| Parameter         | Description                       |
+| ---------         | ----------------------------------|
+| networkId         | Internal blink network ID         |
+| cameraId          | Internal blink camera ID          |
+| cameraType        | Type of camera: CAMERA or OWL (blink mini) |
+
+Besides the channels, the current thumbnail is also provided by a servlet. The url is set in the thing properties.
+![](doc/thumbnail-url.png)
+
+### blink Network
+
+A blink network basically corresponds to the blink sync modules and groups cameras. A blink network can be armed,
+activating all cameras which have motion detection enabled. Cameras with motion detection in a disarmed network won't
+trigger alerts.
+
+Configuration parameters should be set by auto-discovery after configuration of an account exclusively (i.e. no manual
+setup). For completeness, the configuration parameters set by auto-discovery are:
+
+| Parameter         | Description                       |
+| ---------         | ----------------------------------|
+| networkId         | Internal blink network ID         |
+
+## Channels
+
+### blink Camera
+
+| channel  | type   | description                  |
+|----------|--------|------------------------------|
+| motiondetection  | Switch | Enables/disables motion detection for this camera.  |
+| battery | LowBattery | Read-only channel, triggering ON when battery status is low. Not available in blink minis. |
+| temperature | Number | Read-only channel, outputting camera temperature. Not available in blink minis. |
+| setThumbnail | Switch | Write-only channel, triggering taking a new snapshot as thumbnail. Also triggers a new getThumbnail state on completion. |
+| getThumbnail | Image | Read-only channel, returns the current thumbnail. Triggers a state change on new thumbnail.  |
+| motionTriggered | Trigger | Triggered when motion event is detected during account refresh. Therefore, this is linked to the refresh interval of the account and will not be in real-time.|
```

Comment:
The channel motionTriggered is a trigger channel with no data, therefore has no type at all. Since trigger channels are not supposed to be linked to items anyways, what type should be stated in the documentation? Should we leave the column empty?

Reply:
You're right, "Trigger" is absolutely fine!